app/helpers/blacklight_helper.rb

```diff
@@ -10,9 +10,11 @@ def document_partial_name(document, base_name = nil)
 
     display_type ||= document.display_type
 
+    display_type ||= document['eds_publication_type']
```

Comment:
👍 Updating to make our override closer to BL's sounds good (if I'm understanding correctly)